src/lib/isIdentityCard.js

```diff
@@ -75,6 +75,11 @@ const validators = {
 
     return c === 0;
   },
+  IT: function IT(str) {
+    if (str.length !== 9) return false;
+    if (str === 'CA00000AA') return false; // https://it.wikipedia.org/wiki/Carta_d%27identit%C3%A0_elettronica_italiana
+    return str.search(/C[A-Z]\d{5}[A-Z]{2}/is) > -1
```

Comment:
This line requires a semicolon for the linter to allow it. Please run `npm run test --fix` to automatically resolve these types of issues in the future.